Makefile

```diff
@@ -0,0 +1,16 @@
+SPARK_IMAGE_NAME ?= mesosphere/spark-2.4.4-bin-hadoop2.7-k8s
+SPARK_ON_K8S_OPERATOR_IMAGE_NAME ?= mesosphere/spark-on-k8s-operator
+SPARK_ON_K8S_DOCKER_FILE_PATH ?= images/operator/Dockerfile
+
+.PHONY: spark-build
+spark-build:
+	docker build -t ${SPARK_IMAGE_NAME} images/spark
+
+.PHONY: docker-build
+docker-build:
+	docker build -t ${SPARK_ON_K8S_OPERATOR_IMAGE_NAME} \
+	-f ${SPARK_ON_K8S_DOCKER_FILE_PATH} shared/spark-on-k8s-operator
```

Comment:
can you please rebase on the latest master because this path `shared/spark-on-k8s-operator` is just `spark-on-k8s-operator` now?